crates/relayer/src/worker/packet.rs

```diff
@@ -92,6 +104,48 @@ pub fn spawn_packet_cmd_worker<ChainA: ChainHandle, ChainB: ChainHandle>(
     })
 }
 
+pub fn spawn_incentivized_packet_cmd_worker<ChainA: ChainHandle, ChainB: ChainHandle>(
+    cmd_rx: Receiver<WorkerCmd>,
+    // Mutex is used to prevent race condition between the packet workers
+    link: Arc<Mutex<Link<ChainA, ChainB>>>,
+    path: Packet,
+    fee_filter: FeesFilters,
```

Comment:
The naming of the variable and its type is a bit inconsistent, let's harmonize that after renaming the type as described above.